fhir-bucket/src/main/java/com/ibm/fhir/bucket/persistence/AllocateJobs.java

```diff
@@ -73,6 +79,21 @@ public void run(IDatabaseTranslator translator, Connection c) {
             logger.log(Level.SEVERE, NEXTVAL, x);
             throw new DataAccessException("Get next allocationId failed");
         }
+
+        // Build the bucket-path predicate if the user has specified any
+        StringBuilder bpBuilder = new StringBuilder();
+        for (@SuppressWarnings("unused") BucketPath bp: bucketPaths) {
```

Comment:
This is a bit strange.

Reply:
I don't need to use the value here, I just need to create a predicate for each entry. I could get the size and do a for count loop...but then the loop variable wouldn't really be used either (other than to terminate the loop), but this is just as easy.

Reply:
I understand, it's just funky.